### PR TITLE
Adds Lotus holoparasite color & Assassin holoparasite stealth toggle button

### PR DESCRIPTION
**Pull request by ChangelingRain**
Tellurium Lotus
It's black. It's a black lotus.

Assassin holoparasites now have a unique stealth button instead of a default mode toggle button.
Holoparasites with no mode will have a greyed out toggle button.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
Both edgy and tactical

**Comment by Armhulen** [comment]
good/10

**Comment by KorPhaeron** [comment]
banned in tourneys sorry

**Comment by KazeEspada** [comment]
>not white lotus

>not red lotus

**Comment by KorPhaeron** [comment]
>doesn't compile

Also I dislike the way

Petunia/Daisy/Zinnia/Peony sound. Lets brainstorm edgier/cooler sounding flowers

**Comment by bobdobbington** [comment]
should be able to sacrifice this to get TC tbh

File: code/_onclick/hud/guardian.dm [hunk 1]
```diff
@@ -14,7 +14,13 @@
...
 	static_inventory += using
 
-	using = new /obj/screen/guardian/ToggleMode()
+	switch(owner.type)
+		if(/mob/living/simple_animal/hostile/guardian/ranged,/mob/living/simple_animal/hostile/guardian/healer)
+			using = new /obj/screen/guardian/ToggleMode()
+		if(/mob/living/simple_animal/hostile/guardian/assassin)
+			using = new /obj/screen/guardian/ToggleMode/Assassin()
+		else
+			using = new /obj/screen/guardian/ToggleMode/Inactive()
```
Comment (duncathan) [inline code/_onclick/hud/guardian.dm]: Surely you could make this a var on holos and then replace this whole switch with `using = new toggle_path()`

**Pull request merged**